### PR TITLE
Merge upstream 1.10.8

### PR DESCRIPTION
**Pull request by hbandura**
The removal of Celo66 (Eth65) was already merged previously. In 1.10.8 this is reverted due to a hotfix, but here it is not, since it's not needed.

### CODE REVIEW

**Comment by piersy** [comment]
Coverage from tests in `./e2e_test/...` for `./consensus/istanbul/...` at commit 635df612207d906746b1549319bb337793f49d35
<details><summary>coverage:  51.0% of statements across all listed packages</summary><br>
<pre>
coverage:  64.9% of statements in consensus/istanbul
coverage:  41.8% of statements in consensus/istanbul/announce
coverage:  54.8% of statements in consensus/istanbul/backend
coverage:   0.0% of statements in consensus/istanbul/backend/backendtest
coverage:  24.3% of statements in consensus/istanbul/backend/internal/replica
coverage:  69.9% of statements in consensus/istanbul/core
coverage:  45.0% of statements in consensus/istanbul/db
coverage:   0.0% of statements in consensus/istanbul/proxy
coverage:  75.3% of statements in consensus/istanbul/uptime
coverage: 100.0% of statements in consensus/istanbul/uptime/store
coverage:  51.8% of statements in consensus/istanbul/validator
coverage:  79.2% of statements in consensus/istanbul/validator/random
</pre>
CommentID: 9e57b9fb69
</details>

File: internal/ethapi/api.go [hunk 1]
```diff
@@ -1089,14 +1090,15 @@ func FormatLogs(logs []vm.StructLog) []StructLogRes {
 }
 
 // RPCMarshalHeader converts the given header to the RPC output .
-func RPCMarshalHeader(head *types.Header) map[string]interface{} {
+func RPCMarshalHeader(head *types.Header, _ consensus.Engine) map[string]interface{} {
+	miner := head.Coinbase // use the coinbase, not the validator author address for istanbul
```
Comment (gastonponti) [inline internal/ethapi/api.go:1094]: Shouldn't we have to add both? Author and FeeCurrencyAddress?

Comment (hbandura) [reply internal/ethapi/api.go:1094]: Well, it's a decision we must take. @mcortesi what do you think?

Comment (mcortesi) [reply internal/ethapi/api.go:1094]: i would not add it for now. we have enough problems with not backward compatibility issues. (thus i guess being broken already, we could, but probably we need to check everywhere for consistency, not part of this pr though)

**Pull request merged**